setup.py

```diff
@@ -13,6 +14,9 @@
 
 # Key reference followed: https://github.com/pyg-team/pytorch_geometric/blob/master/setup.py
 
+# Set an environment variable to bypass the SKLEARN error message.
```

Comment:
Please add the link to [sklearn-pypi-package](https://github.com/scikit-learn/sklearn-pypi-package#how-to-fix-the-error-for-the-main-use-cases) for further information, preventing those unfamiliar with this PR from getting confused about why this error was bypassed.